src/modules/Elsa.Workflows.Runtime/Services/DefaultWorkflowDefinitionStorePopulator.cs

```diff
@@ -49,23 +49,37 @@ public class DefaultWorkflowDefinitionStorePopulator : IWorkflowDefinitionStoreP
     }
 
     /// <inheritdoc />
-    public async Task PopulateStoreAsync(CancellationToken cancellationToken = default)
+    public Task PopulateStoreAsync(CancellationToken cancellationToken = default)
+    {
+        return PopulateStoreAsync(true, cancellationToken);
+    }
+
+    /// <inheritdoc />
+    public async Task PopulateStoreAsync(bool indexTriggers, CancellationToken cancellationToken = default)
     {
         var providers = _workflowDefinitionProviders();
         foreach (var provider in providers)
         {
             var results = await provider.GetWorkflowsAsync(cancellationToken).AsTask().ToList();
 
-            foreach (var result in results) await AddAsync(result, cancellationToken);
+            foreach (var result in results) await AddAsync(result, indexTriggers, cancellationToken);
         }
     }
 
     /// <inheritdoc />
-    public async Task AddAsync(MaterializedWorkflow materializedWorkflow, CancellationToken cancellationToken = default)
+    public Task AddAsync(MaterializedWorkflow materializedWorkflow, CancellationToken cancellationToken = default)
```

Comment:
Why not give indexTriggers in the next method a default of `true`? Than this method would not be needed

Reply:
What about callers that want the default of `true`, but not the default of `CancellationToken`? They would have to do this:

```csharp
await populator.AddAsync(MaterializedWorkflow materializedWorkflow, cancellationToken: cancellationToken);
```

With this convenient overload, they can now do this:

```csharp
await populator.AddAsync(MaterializedWorkflow materializedWorkflow, cancellationToken);
```